docs/getting-started/install-configure.md

```diff
@@ -1,20 +1,54 @@
 ---
-title: Install & Configure
-toc: true
-weight: 2
-indent: true
+title: Install & Configure toc: true weight: 2 indent: true
 ---
 
-# Install & Configure Crossplane
+# Choosing Hosted or Self-Hosted Crossplane
 
-Crossplane can be easily installed into any existing Kubernetes cluster using
-the regularly published Helm chart. The Helm chart contains all the custom
-resources and controllers needed to deploy and configure Crossplane.
+Users looking to use Crossplane for the first time have two options avaiable to
+them today. The first way is to use a hosted Crossplane service like [Upbound
+Cloud]("https://upbound.io"). Alternatively, users looking for some more
```

Comment:
Nit: This doc uses reference style links - it would be good to be consistent.

https://www.markdownguide.org/basic-syntax/#reference-style-links

Reply:
Done.